### ISSUE TITLE
Hide SSH password/passphrase

### ISSUE THREAD

**Issue by bbova** [fabianfiorotto/quick-query#57]
Love the ability to connect via SSH now!  

One recommendation is to hide the password and passphrase in the UI, possibly with just **********

**Comment by fabianfiorotto** [comment]
To implement this It would require to switch to HTML inputs.
When I saw that they added HTML inputs to the style guide I decided to replace the mini editors on the connect view with inputs. But back in the day, the support for HTML inputs was terrible, the connect view looked ugly. Now I tried again and I see that they look almost the same

Atom light theme: html inputs on the left mini editors on the right.
![light](https://user-images.githubusercontent.com/898842/51246577-7e231680-1969-11e9-9674-1634c02a72ec.png)

One dark theme: html inputs on the left mini editors on the right.
![dark](https://user-images.githubusercontent.com/898842/51246742-d8bc7280-1969-11e9-80a4-c7d867f3246d.png)

The thing that left to test is community themes. The last time I tried they were unusable with html inputs. I should take the 20 most used themes and start to try. I know that there are a lot of themes that are unmaintained but still used and I don't want to left anybody out.
